template/7-functional_requirements.md

```diff
@@ -1,22 +1,21 @@
 # Functional Requirements
 
-*Max 3 pages.*
+1. **User Authentication**: Users must be able to authenticate into their account to user the app. For this, we use pre-existing infrastructure, such as google authentication.
 
-*List the key features of the MVP precisely.*
+2. **Post Creation**: Users must be able to create posts containing text. These posts will have embedded location data.
 
-- User Authentication
-- Post Creation
-- Location-Based Feed
-- Commenting System
-- Offline Mode
-- Voting System
-- Posts and comments management
-- Centauri Points
-- Challenges
-- Map Integration
+3. **Location-Based Feed**: Users must be able to see in a feed all of the posts that were made inside of a 100m radius around their current position. This feed will have different sorting options.
 
+4. **Commenting System**: Users must be able to leave comments on posts. 
```

Comment:
and see most recent comments first by default or most popular first if they prefer